specs/waku/v2/waku-message.md

```diff
@@ -76,6 +77,10 @@ topic, data, nonce`.
 Since Waku v2 is using libp2p PubSub, some of these fields can be dropped. The previous `topic`
 field corresponds to `contentTopic`. The previous `data` field corresponds to the `payload` field.
 
+# Security Consideration
+
+In Waku v2, the confidentiality, integrity, and authenticity of the data must be addressed at the `WakuMessage` level.  That is, the `payload` shall be encrypted or signed properly to meet the application-specific privacy needs.  Notice that due to the anonymity considerations these security features are not supported by the `WakuRelay` protocol and are delegated to the upper levels.
```

Comment:
Not sure we need to mention relay like this here, for the context of this spec we are in Waku Message spec. Aren't some properties provided for with version 1?

Reply:
> Aren't some properties provided for with version 1?

As far as the specs show, in waku v1, envelopes can be signed then encrypted
https://github.com/vacp2p/specs/blob/master/specs/waku/v1/waku-1.md#payload-encryption
https://github.com/vacp2p/specs/blob/master/specs/waku/v1/envelope-data-format.md#signature

Reply:
> Not sure we need to mention relay like this here, for the context of this spec we are in Waku Message spec

Sure, will remove it